packages/installations/src/helpers/buffer-to-base64-url-safe.test.ts

```diff
@@ -19,8 +19,13 @@ import { expect } from 'chai';
 import '../testing/setup';
 import { bufferToBase64UrlSafe } from './buffer-to-base64-url-safe';
 
-const TYPED_ARRAY_REPRESENTATION = new TextEncoder().encode('hello world');
-const BASE_64_REPRESENTATION = btoa('hello world');
+const str = 'hello world';
+const TYPED_ARRAY_REPRESENTATION = new Uint8Array(str.length);
```

Comment:
Changed so it works in IE11 without TextEncoder polyfill, though there still are many tests failing in IE11 (due to indexedDB?). Is installations designed to work in IE11 ?

Reply:
@mmermerkaya ping

Reply:
Installations should work in IE11. I have not received any bug reports about it not working in IE11.